client/lang/src/en.json

```diff
@@ -8,5 +8,7 @@
     "CampaignAdmin.NO_CHANGES": "No changes",
     "CampaignAdmin.PUBLISHCAMPAIGN": "Publish campaign",
     "CampaignAdmin.REMOVED": "Removed",
-    "CampaignAdmin.REVERTCAMPAIGN": "Revert"
-}
\ No newline at end of file
+    "CampaignAdmin.REVERTCAMPAIGN": "Revert",
+    "CampaignAdmin.PUBLISH_FAIL": "Campaign could not be published.",
```

Comment:
@maxime-rainville What does a page do when it fails publishing?

Reply:
The expectation here is that the back end will give us a proper error message and we will display that.

This string is only here in case we don't get an error message back and we don't know why publication failed.